packages/nimble-components/src/table-column/enum-text/cell-view/index.ts

```diff
@@ -0,0 +1,61 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { styles } from '../../text-base/cell-view/styles';
+import { template } from '../../text-base/cell-view/template';
+import type { TableColumnEnumCellRecord } from '../../enum-base';
+import type { MappingConfigText } from '../../../mapping/text';
+import { TableColumnTextCellViewBase } from '../../text-base/cell-view';
+import type { TableColumnEnumTextColumnConfig } from '..';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-cell-view': TableColumnEnumTextCellView;
+    }
+}
+
+/**
+ * A cell view for displaying mapped text
+ */
+export class TableColumnEnumTextCellView extends TableColumnTextCellViewBase<
+TableColumnEnumCellRecord,
+TableColumnEnumTextColumnConfig
+> {
+    public override get text(): string {
+        return this.getMappingToRender()!.label;
```

Comment:
Seeing `!` here makes me nervous. Do we have assurances that `this.getMappingToRender()` will return a non-null value even though the return type isn't specified as that? It doesn't seem like that's the case since the getter for `shouldUsePlaceholder` is based on `this.getMappingToRender()` returning `null`.

I think it would be safer to write this as
```suggestion
        return this.getMappingToRender()?.label ?? '';
```

Reply:
This was based on the knowledge that the base class only ever calls the `text` getter when `shouldUsePlaceholder` is `false`. In my case, `shouldUsePlaceholder` being false guarantees that `getMappingToRender()` returns non-null. However, maybe it's better not to rely on that knowledge of the base class.

Reply:
I agree that we shouldn't rely on that knowledge in the base class, especially because this is a public getter, so there isn't anything guaranteeing it wouldn't be called in more places or under different circumstances.